component-catalog/src/Examples/QuestionBox.elm

```diff
@@ -57,7 +59,18 @@ example =
             , QuestionBox.correct
             ]
         ]
-    , about = []
+    , about =
+        [ Text.smallBody
+            [ Text.html
+                [ text "Learn more about this component from "
+                , ClickableText.link "Tessa's demo"
+                    [ ClickableText.linkExternal "https://www.dropbox.com/preview/NRI%20Engineering/Demos/2022-12-22%20-%20Tessa%20-%20QuestionBox%20and%20Block.mp4?role=work"
```

Comment:
I'm not sure that we should link to this demo for QuestionBox, as the question box has changed a _lot_ since this point.

In my demo:
<img width="398" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/6e56d47d-2a6b-43a5-a906-82f409a89fda">

Currently:
<img width="745" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/59af4eed-bc32-4a66-8156-8ef9b2d51154">

Reply:
Fine with me - I didn't review all of the demos. (If you are able to make the change to remove it, I would appreciate that!)